_rules/aria-required-owned-element-bc4a75.md

```diff
@@ -24,16 +24,13 @@ acknowledgments:
 
 ## Applicability
 
-This rule applies to any [HTML or SVG element][] that is [included in the accessibility tree][] and has a [WAI-ARIA 1.1][] [explicit semantic role][] with [required owned elements][], except if one or more of the following is true:
-
-- the element has a [semantic role][] of `combobox`; or
-- the element has the `aria-busy` [attribute value][] of `true`, or has an [ancestor][] in the accessibility tree with this [attribute value][].
+This rule applies to any [HTML or SVG element][] that is [included in the accessibility tree][] and has a [WAI-ARIA 1.2][] [explicit semantic role][] with [required owned elements][], except if the element has the `aria-busy` [attribute value][] of `true`, or has an [ancestor][] in the accessibility tree with this [attribute value][].
```

Comment:
I prefer to be explicit on the atribute that needs to have the value of `true`, otherwise it could be interpreted as an ancestor who has any attribute with the attribute value of true (because only the attribute value is specified in the current formulation).

```suggestion
This rule applies to any [HTML or SVG element][] that is [included in the accessibility tree][] and has a [WAI-ARIA 1.2][] [explicit semantic role][] with [required owned elements][], except if the element has the `aria-busy` [attribute value][] of `true`, or has an [ancestor][] in the accessibility tree with the `aria-busy` [attribute value][] of `true`.
```